mantidproject/mantid
# Isolate Qwt-related parts of instrument view

**Description of work.**

Refactors the Qwt-related functionality ready for defining separate matplotlib-based classes for the Qt5 build of the instrument view. Functionally nothing has changed.

**Report to:** [nobody]. 

**To test:**

* Code review
* The instrument view functionality should be entirely unchanged.

Refs #20240

*This does not require release notes* because **this is an internal change.**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
